dask_sphinx_theme/ext/dask_config_sphinx_ext.py

```diff
@@ -1,11 +1,16 @@
+import dask_sphinx_theme
 import requests
 import yaml
 from docutils import nodes
 from docutils.parsers.rst import Directive, directives
 
 
 def get_remote_yaml(url):
-    r = requests.get(url)
+    r = requests.get(
+        url,
+        headers={"User-Agent": f"Dask sphinx theme {dask_sphinx_theme.__version__}"},
+    )
+    r.raise_for_status()
```

Comment:
Note that this line isn't actually needed, but makes debugging things much easier. Right now we get a weird yaml parsing error instead of an HTTP error.